frontend/src/components/sidebar/MenuItemComponent.js

```diff
@@ -11,24 +11,25 @@ const useStyles = createUseStyles({
     },
     container: {
         display: 'flex',
-        height: 56,
+        height: 40,
         cursor: 'pointer',
         '&:hover': {
             backgroundColor: ({ theme }) => theme.color.paleBlueTransparent
         },
-        paddingLeft: ({ level }) => 32 * level,
+        paddingLeft: ({ level }) => 16 * level,
         transition: 'all 0.2s ease-in-out'
     },
     leftBar: {
         borderLeft: ({ theme, level }) =>
             level > 1 ? 'none' : `3px solid ${theme.color.darkGrayishBlue}`
     },
     title: { 
-        fontSize: 15, // left bar titles 
-        lineHeight: '25px',
+        fontSize: ({ theme }) => theme.typography.icontitle.fontSize , // left bar titles 
+        lineHeight: '20px',
         letterSpacing: '0.2px',
-        color: ({ theme, isActive }) =>(isActive ? '#00b300' : '#b5b4b9'), //(isActive ? theme.color.paleBlue : theme.color.grayishBlue),
-        marginLeft: 20
+        color: ({ theme, isActive }) =>(isActive ? theme.uniformStyle.color.highlightingColor : theme.uniformStyle.color.secondaryFontColor), //(isActive ? theme.color.paleBlue : theme.color.grayishBlue),
+        marginLeft: 10,
+        fontWeight:'inherit'
     }
 });
 
```

Comment:
I don't quite understand this file, maybe you could add a little documentation Comment at the top @IremToroslu

Reply:
sure @Waldleufer I will document it again! thanks for your feed back and suggestions. Appreciate it :)